### DISCUSSION TITLE
Caso d'uso di nascita mancante, per una delle combinazioni, per bimbo nato vivo ma morto prima della denuncia di nascita.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#1020] [Miglioramenti]
Buonasera,

non riusciamo a trovare il caso d'uso corrispondente al caso di 'Dichiarazione nascita: Dichiarazione nei termini di filiazione fuori dal matrimonio resa all'USC dal padre con consenso materno espresso con atto notarile di bimbo nato vivo e deceduto prima della dichiarazione di nascita'.

Per la combinazione indicata di dichiarante e consenso, ci sono i casi d'uso, per il 'bimbo nato vivo':

ID:11211004 - 'Dichiarazione nascita: Dichiarazione entro i 10 giorni - resa dal padre con il consenso della madre reso mediante atto notarile - filiazione fuori dal matrimonio';

per il 'bimbo nato morto':

ID:11211104 - 'Dichiarazione nascita: Dichiarazione nei termini di filiazione fuori dal matrimonio resa all'USC dal padre con consenso materno espresso con atto notarile di bimbo nato morto';

mentre non troviamo il caso del 'bimbo nato vivo e deceduto prima della dichiarazione di nascita', che secondo la logica di assegnazione di ID utilizzata, dovrebbe avere ID: 11211204, che appunto manca.

Anche a voler utilizzare uno degli altri 2 casi d'uso, censiti, passando la corretta valorizzazione dei flag, 'natoMorto' = 'NO', e 'natoVivoMaDeceduto' = 'SI', ci viene restituita un'anomalia di pattern non rispettato sul campo 'natoVivoMaDeceduto', in quanto è richiesto obbligatoriamente il valore 'NO'.

Potete fare una verifica?  

In attesa di riscontro,

cordiali saluti.

**Comment by acolosimosogei** [comment]
Buongiorno, stiamo lavorando alla discussion, le daremo un riscontro quanto prima.
Grazie.

**Comment by asqueglia** [comment]
Con il rilascio della versione 1.34.0 la segnalazione è stata risolta
